src/test/java/core/basesyntax/handler/PurchaseOperationTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.handler;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class PurchaseOperationTest {
+    private static OperationHandler purchase;
+
+    @BeforeAll
+    static void beforeAll() {
+        purchase = new PurchaseOperation();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.storage.clear();
+    }
+
+    @Test
+    void purchaseOk() {
+        Storage.storage.put("peach", 10);
+        purchase.handle(Storage.storage,
+                new FruitTransaction(FruitTransaction.Operation.PURCHASE,"peach",5));
```

Comment:
```suggestion
                new FruitTransaction(FruitTransaction.Operation.PURCHASE, "peach", 5));
```
Fix whitespaces everywhere